platform/net/fakes/fake_routes_searcher.go

```diff
@@ -9,6 +9,6 @@ type FakeRoutesSearcher struct {
 	SearchRoutesErr    error
 }
 
-func (s *FakeRoutesSearcher) SearchRoutes() ([]boshnet.Route, error) {
+func (s *FakeRoutesSearcher) SearchRoutes(ipv6 bool) ([]boshnet.Route, error) {
```

Comment:
can we choose `ipv6`, `isIpv6` (or perhaps `isIPv6`)... 

`is_ipv6` seems wrong in that Golang suggest camelCase for variables, as does `IsVersion6` (see `func (ia failingInterfaceAddress) GetIP(IsVersion6 bool) (string, error)`)